README.md

```diff
@@ -71,6 +71,34 @@ Before you can run the PerfKit Benchmaker on Cloud providers you need accounts a
 You also need the software dependencies, which are mostly command line tools and credentials to access your
 accounts without a password.  The following steps should help you get the CLI tool auth in place.
 
+If you are running on Windows, you will need to install Github Windows
```

Comment:
Yes a windows controller can run benchmarks on linux guests - that's all this PR enables - windows guests will come later.